packages/cadl-python/src/emitter.ts

```diff
@@ -222,6 +238,13 @@ function emitBodyParameter(program: Program, bodyType: Type, params: HttpOperati
     } else {
         type = getType(program, bodyType);
     }
+
+    // avoid anonymous model type
+    if (type && !type.name) {
+        type.name = operation.container.name + operation.operation.name[0].toUpperCase() + operation.operation.name.slice(1) + "Request";
+        type.snakeCaseName = camelToSnakeCase(type.name);
+    }
```

Comment:
This may generate good name but it shall generate unique name to avoid anonymous model name